### PR TITLE
MM-28448 - Fix cypress test

### PR DESCRIPTION
**Pull request by cpoile**
#### Summary
- it's tough to use class names as we move to styled-components, so I'm proposing to put 'data-testid' tags on any element we want to reference

#### Ticket Link
- https://mattermost.atlassian.net/browse/MM-28448

### CODE REVIEW

File: tests-e2e/cypress/integration/backstage/incident_list_spec.js [hunk 1]
```diff
@@ -76,6 +78,6 @@ describe('backstage incident list', () => {
         });
 
         // * Verify that the header contains the incident name
-        cy.get('.details-header').contains(incidentName);
+        getTestElement('incident-title').contains(incidentName);
```
Comment (lieut-data) [inline tests-e2e/cypress/integration/backstage/incident_list_spec.js:81]: I think there is already `cy.findByTestId` for this?

**Review by lieut-data — APPROVED** [review]
Thanks! Just one comment below.

File: tests-e2e/cypress/integration/backstage/incident_list_spec.js (hunk 1 above)
Comment (cpoile) [reply tests-e2e/cypress/integration/backstage/incident_list_spec.js]: 🤦 Hah, didn't know about that. Thanks!

**Review by lieut-data — APPROVED** [review]

**Review by prapti — APPROVED** [review]
LGTM! Thank you @cpoile!

**Pull request merged**